app/views/meetings/show.html.erb

```diff
@@ -43,12 +43,31 @@
   </div>
   </div>
 
+
   <% if @meeting_owner == true %>
   <div class="row">
     <br>
     <br>
     <h2>Invite Employees</h2>
+    <p><strong>Company: </strong><%= link_to @invitees[0].company.name, @invitees[0].company %> </p>
     <% if @invitees.count > 0 %>
+        <div class="well center" style="width:50%">
+            <input type='text' id='search' placeholder="name" />
+            <script>
+              $("#search").on("keyup paste change", function(){
+                  var re = new RegExp(this.value,"i");
+                  $("#invitees tr").each(function(index){
+                      var invitee_name = $("#invitee_name", this).html();
+                      this.hidden = ! re.test(invitee_name);
+                  });
```

Comment:
What does this code do?

Also: We should not have any javascript inside our views, this needs to go in a `.js` file in assets.